loky/backend/spawn.py

```diff
@@ -172,23 +168,21 @@ def prepare(data, parent_sentinel=None):
     if "orig_dir" in data:
         process.ORIGINAL_DIR = data["orig_dir"]
 
-    if "mp_tracker_args" in data:
+    if "mp_tracker_fd" in data:
         from multiprocessing.resource_tracker import (
             _resource_tracker as mp_resource_tracker,
         )
 
-        mp_resource_tracker._fd = data["mp_tracker_args"]["fd"]
-        mp_resource_tracker._pid = data["mp_tracker_args"]["pid"]
-    if "tracker_args" in data:
+        mp_resource_tracker._fd = data["mp_tracker_fd"]
```

Comment:
I think there you have to explicitly set `mp_resource_tracker._pid` to None also ?

That might be the reason the windows pipeline is still failing. Skimming through `popen_loky_win32.py` I wonder if `mp_resource_tracker._pid` is not inherited from the parent and thus can have a non-null value.

Same for `_resource_tracker` in the following blocks.

Reply:
In `spawn`, I would say there is no inheritance so the `pid` should be initialized to `None` here:
https://github.com/python/cpython/blob/main/Lib/multiprocessing/resource_tracker.py#L67

From what I see, this looks like a case where we call the stop function twice? But hard to debug without having access to a windows machine (which I can't before next week).